yang/frr-bgp.yang

```diff
@@ -1090,6 +1111,26 @@ module frr-bgp {
     uses structure-neighbor-group-filter-config;
   }
 
+  augment "/frr-rt:routing/frr-rt:control-plane-protocols/frr-rt:control-plane-protocol/bgp/neighbors/unnumbered-neighbor/afi-safis/afi-safi/linkstate-linkstate" {
+    uses structure-neighbor-route-reflector;
+
+    uses structure-neighbor-route-server;
+
+    uses structure-neighbor-group-soft-reconfiguration;
+
+    uses structure-neighbor-group-filter-config;
+  }
+
+  augment "/frr-rt:routing/frr-rt:control-plane-protocols/frr-rt:control-plane-protocol/bgp/neighbors/unnumbered-neighbor/afi-safis/afi-safi/linkstate-linkstate" {
+    uses structure-neighbor-route-reflector;
+
+    uses structure-neighbor-route-server;
+
+    uses structure-neighbor-group-soft-reconfiguration;
+
+    uses structure-neighbor-group-filter-config;
+  }
+
```

Comment:
duplicate augment.
adding to this, I presume this was for linkstate-vpn.
same comment as https://github.com/FRRouting/frr/pull/12649/commits/dfdf3f538e289726099209730eb27f66cc9d156f#r1176079790

Reply:
removed